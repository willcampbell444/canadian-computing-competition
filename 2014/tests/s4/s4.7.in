100
100
0 0 1000000000 1000000000 5
49846164 115537078 61082450 245839212 2
196377496 627188412 305884201 712893868 7
315611185 140441492 957796030 192561399 8
280037314 95960753 804610403 382501256 18
143097394 3683157 211767899 595653992 9
7851024 394148218 609524996 582590160 22
427545585 71342811 555728029 507581178 16
289325551 584537427 375829143 810983768 11
551182969 200393441 999721513 280685404 10
160402509 585339848 246824316 792974972 8
131304394 367227682 920628475 383085884 1
543378824 112922856 666363684 240196461 21
736420537 71690891 963457649 805991557 14
49230797 433779662 156009582 768490266 23
269420140 248748297 910826204 992558006 24
7737516 633902909 339801990 893011751 18
33001967 51768760 817886621 177597894 4
115484436 539072264 311091467 769165804 9
856964570 577614655 982818400 978174401 14
117934854 74527759 554088394 589658960 16
540375403 174033636 903571617 471282175 18
72943235 779228217 418132489 891255048 17
571422666 353861272 964163155 926012663 11
85601486 476105675 345934172 670374953 7
192015350 284571676 227721036 327044588 17
88213071 172743840 422764512 271858184 2
68199637 268010140 173198414 898380991 9
68281000 280989819 308186867 759095684 24
203680246 68915266 323133670 164944398 25
317802467 514215974 953662684 538255620 5
96609440 217204142 833080953 791177210 10
607042966 23789504 984209349 332974381 1
674586320 206090920 973974310 465853929 8
223366287 108749231 967860523 436162615 24
146400292 680915022 968894133 982664970 20
108451411 278946016 272185764 676715214 15
64526101 51696016 212932675 596911735 15
34584676 89559774 992551911 959976860 3
51128312 320064178 506425787 813487626 2
10780848 256649756 946571222 805069182 18
157095457 45926829 796652777 930150834 9
221459217 329237451 387823474 475883062 25
519956422 361641322 555638755 589603489 19
9669458 5438865 78162638 891797398 17
157223975 58445208 681562755 140428765 6
109274829 276511396 607614867 898311378 18
222745809 87507776 380504929 417793720 1
542404188 912084780 670608368 920644714 7
154782262 133565580 458960395 822764111 13
274272431 917103851 414422954 989819933 1
662923336 255342552 696519053 400326094 11
180215748 238139412 608354543 934117560 5
403212908 133933403 433145595 701430414 9
65702432 247158644 982623058 796473570 14
63842973 60002290 288640780 813009466 17
644025283 413034968 810165965 608329079 1
42918866 575854049 557636415 576574591 5
180071440 22962703 647568103 656216651 6
199423810 543303380 468273808 641468543 15
707554633 46741969 972392981 658261191 20
189191582 112616046 360617113 485944935 9
6592750 75661830 924031749 392848918 17
305743457 81805337 324177714 494893107 12
139329226 742104730 762153913 928328276 6
384651963 611545273 513173433 919747865 18
343405600 248922566 460206583 329342904 13
592648322 154380858 625135027 611176262 9
580828154 311103595 742907560 370415840 6
730838685 392296534 923330365 696594527 17
317613882 285146911 530541472 458400754 14
797998919 67978943 909250231 725434696 1
30868017 164236414 211346594 693693109 22
50300110 78370431 70708619 634281320 25
38716640 23006633 125407565 43470441 11
722097738 371145123 908105994 659571947 17
360873204 52105743 919135003 702103694 9
770318155 351301901 898487698 925201422 6
385499435 130824240 674947886 633475126 18
22491824 559451771 72097355 855942675 13
247620345 79297368 749493153 739816009 12
108188149 87019966 288342038 547038074 15
569163940 757532709 917456425 839154420 20
774371562 111924907 988975893 194389065 6
611000072 310861292 821228969 958876990 21
93376648 363755558 901187137 843027847 14
442248009 234699228 681568364 421368190 11
16178337 119349014 735125963 581617528 6
690592736 75643394 867537135 790468757 3
294641149 110448803 383513351 729479840 17
126408990 133058364 843631535 228154444 15
596585809 9522598 638466693 526903771 12
334581159 377859585 615773700 986330045 20
392476172 379388782 928226371 424726060 10
799647194 118808352 802678116 510705210 15
846315103 26720617 855301659 592428374 20
755942978 391131621 926949338 413354172 9
61999071 281796080 949955358 506253301 4
410083814 3621868 459091134 35928661 14
342434751 1714835 393357732 44669291 3
