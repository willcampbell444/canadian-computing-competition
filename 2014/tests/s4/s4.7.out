669680334382539317
